doc/images.md

```diff
@@ -148,6 +148,24 @@ A `SpriteBatchComponent` is also available for your convenience.
 
 See example [here](https://github.com/flame-engine/flame/tree/master/doc/examples/sprite_batch).
 
+## Composition
+
+In some cases you may want to merge multiple images into a single image, this is called [Compositing](https://en.wikipedia.org/wiki/Compositing). This can be useful for example when working with the [SpriteBatch](#spritebatch) API to optimize your drawing calls. 
+
+For such usecases Flame comes with the `Composition` class. This allows you to add multiple images, each at their own position, onto a new image:
+
+```dart
+final composition = Composition()
+  ..add(image1, Vector2(0, 0))
+  ..add(image2, Vector2(64, 0));
+  ..add(image3, 
+    Vector2(128, 0), 
+    source: Rect.fromLTWH(32, 32, 64, 64),
+  );
+
+Image image = await composition.compose();
+```
```

Comment:
should we mention that this is too expensive to do every tick, and therefore compositions must be pre-rendered?